src/schema/schema.graphql

```diff
@@ -1268,6 +1639,7 @@ type Mutation {
 	modifyAppointment(appointment: CalendarItemInput!): Boolean
 	modifyIdentity(id: ID!, attrs: IdentityAttrsInput!): String
 	modifyPrefs(prefs: PreferencesInput!): Boolean
+	modifyFilterRules(filters: [FilterInput!]): Boolean
```

Comment:
If we have FilterInput as required, is it possible to delete all of your filter rules?

Reply:
Yes
This problem will arrive only if we had used [FilterInput]! where it will not allow empty array
but we used [FilterInput!] so it will allow empty array but will not allow empty FilterInput object